components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/permission/GeckoPermissionRequest.kt

```diff
@@ -66,7 +67,8 @@ sealed class GeckoPermissionRequest constructor(
                 PERMISSION_AUTOPLAY_AUDIBLE to Permission.ContentAutoPlayAudible(),
                 PERMISSION_AUTOPLAY_INAUDIBLE to Permission.ContentAutoPlayInaudible(),
                 PERMISSION_PERSISTENT_STORAGE to Permission.ContentPersistentStorage(),
-                PERMISSION_MEDIA_KEY_SYSTEM_ACCESS to Permission.ContentMediaKeySystemAccess()
+                PERMISSION_MEDIA_KEY_SYSTEM_ACCESS to Permission.ContentMediaKeySystemAccess(),
+                PERMISSION_STORAGE_ACCESS to Permission.ContentStorageAccess()
```

Comment:
nit: I was a bit afraid that name could be easily confused with the `ContentPersistentStorage`. I was thinking about a more descriptive name like  `CrossOriginAccessStorage` what do you think?

Reply:
Indeed they are similar.
I went with using `..StorageAccess` as that is the name in the web API though.
Regarding renaming it, all other permissions start with `Content` so to combine this with your suggestion we'd get `ContentCrossOriginAccessStorage`. Should I change to this?

Reply:
Sounds perfect :)